plugins/UsersManager/lang/en.json

```
@@ -4,150 +4,150 @@
         "TypeYourPasswordAgain": "Type your new password again.",
         "TypeYourCurrentPassword": "Please type your current password to confirm the password change.",
         "User": "User",
         "UserHasPermission": "%1$s currently has %2$s access for %3$s.",
         "UserHasNoPermission": "%1$s currently has %2$s to %3$s",
-        "UsersManagement": "Users Management",
+        "UsersManagement": "User Management",
         "UsersManagementMainDescription": "Create new users or update the existing users. You can then set their permissions above.",
-        "WhenUsersAreNotLoggedInAndVisitPiwikTheyShouldAccess": "When users are not logged in and visit Matomo, they should initially see",
+        "WhenUsersAreNotLoggedInAndVisitPiwikTheyShouldAccess": "When users are not logged in and visit Matomo, they should initially be met by",
         "YourUsernameCannotBeChanged": "Your username cannot be changed.",
-        "YourVisitsAreIgnoredOnDomain": "%1$sYour visits are ignored by Matomo on %2$s %3$s (the Matomo ignore cookie was found in your browser).",
-        "YourVisitsAreNotIgnored": "%1$sYour visits are not ignored by Matomo%2$s (the Matomo ignore cookie was not found in your browser).",
+        "YourVisitsAreIgnoredOnDomain": "%1$sYour visits are ignored by Matomo on %2$s %3$s (the Matomo ignore cookie was found in your web browser).",
+        "YourVisitsAreNotIgnored": "%1$sYour visits are not ignored by Matomo%2$s (the Matomo ignore cookie was not found in your web browser).",
         "AddUserNoInitialAccessError": "New users must be given access to a website, please set the 'initialIdSite' parameter.",
         "AtLeastView": "At least view",
         "ManageUsers": "Manage Users",
         "ManageUsersDesc": "Create new users or update the existing users. You can then set their permissions here too.",
-        "ManageUsersAdminDesc": "Create new users or update the existing users. You can then set their permissions here too. Please note that you are logged in as an administrator. This means you only see and manage users that have access to the specific websites where you have \"admin\" permissions. Only a user with super user access can see and manage all users across all websites.",
-        "NoAccessWarning": "This user has not been granted access to a website. When they login, they will see an error message. To prevent this, add access to a website below.",
+        "ManageUsersAdminDesc": "Create new users or update existing ones and their permissions. As you are logged in as an administrator you only browse and manage users that have access to the specific websites where you have \"admin\" permissions. Only a user with superuser access can browse and manage all users across all websites.",
+        "NoAccessWarning": "This user has not been granted access to a website and will get an error message when logging in. Prevent this by granting access to a website below.",
```

Is this maybe better?
```suggestion
        "NoAccessWarning": "This user has not been granted access to any website and will get an error message when logging in. Prevent this by granting access to a website below.",
```